src/Components/Attributes/Attributes.js

```diff
@@ -105,7 +109,17 @@ class Attributes extends React.Component {
     // `dataStructures` prop is optional and is used to
     // resolve inheritance, references and includes/mixins, plus to
     // render the inheritance tree.
-    const dataStructures = props.dataStructures || [];
+    let dataStructures = props.dataStructures || [];
+    let element = props.element;
+
+    if (isFunction(element.getMetaProperty)) {
```

Comment:
Perhaps a better check would be to see if element is an instance of Element instead of checking if the `getMetaProperty` implementation is a function.

```js
if (element instanceof minim.Element) {
```

Reply:
For this I assumed we'd like to keep attributes-kit codebase `instanceof` use-free.

I'd rather stick to the https://github.com/apiaryio/documentation-service/blob/master/phoenix-ui/src/documentation/utils/minim/predicates/README.md way of detecting things. @char0n So perhaps we are missing a generic `isMinimElement` (which could internally check e.g. the `isFunction(o.getMetaProperty)`) to get to descriptive declarative form?